Gruntfile.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = function (grunt) {
+  require('load-grunt-tasks')(grunt);
+
+  grunt.initConfig({
+    jsFiles: ['./*.js', './lib/**/*.js', './test/**/*.js'],
+    eslint: {
+      target: ['<%= jsFiles %>']
+    },
+    mochaTest: {
+      test: {
+        options: {
+          reporter: 'spec',
+          clearRequireCache: false
+        },
+        src: ['test/*.js']
+      }
+    },
+    watch: {
+      js: {
+        files: ['<%= jsFiles %>'],
+        tasks: ['lint', 'test']
```

Comment:
I think I prefer to run test first so linting errors don't stop broken tests from showing up.